javaparser-testing/src/test/resources/com/github/javaparser/bulk_test_results/javaparser_test_results.txt

```diff
@@ -1,4 +1,2 @@
-javaparser-testing/target/test-classes/com/github/javaparser/TestFileIso88591.java
```

Comment:
Yes, it's the result of the bulk parsing unit test that runs on all JavaParser source code. I fixed it so that it ignores the "target" directory.